hpo_demo_setup.sh

```diff
@@ -241,7 +252,8 @@ function hpo_experiments() {
 		echo
 		echo "Run the benchmark for trial ${i}"
 		echo
-		BENCHMARK_OUTPUT=$(./hpo_helpers/runbenchmark.sh "hpo_config.json" "${SEARCHSPACE_JSON}" "$i" "${DURATION}")
+		BENCHMARK_OUTPUT="Objfunc_result=0.007914818407446147 Benchmark_status=success"
+		#BENCHMARK_OUTPUT=$(./hpo_helpers/runbenchmark.sh "hpo_config.json" "${SEARCHSPACE_JSON}" "$i" "${DURATION}")
```

Comment:
Looks like this has been done by mistake?

Reply:
oops! updated now.